content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -544,6 +546,17 @@ Then, delete the Secret you now know the name of:
 kubectl -n examplens delete secret/example-automated-thing-token-zyxwv
 ```
 
+### Delete/invalidate a short-lived ServiceAccount token {#delete-short-lived}
+
+Short lived ServiceAccount tokens automatically expire after the amount
```

Comment:
```suggestion
Short lived ServiceAccount tokens automatically expire after the time-limit 
```

Reply:
If we are being specific about the expiration then it would be better to use time-limit instead of amount.